packages/react/src/index.ts

```diff
@@ -69,8 +73,8 @@ export type {
 	StackedAreaChartOptions,
 	TreeChartOptions,
 	TreemapChartOptions,
-	WorldCloudChartOptions
+	WordCloudChartOptions
```

Comment:
we should still export both and mark as deprecated

Reply:
Updated